src/pomodoro/containers/PomodoroControls.js

```diff
@@ -74,40 +74,68 @@ class PomodoroControl extends React.PureComponent {
   }
 }
 
-const PomodoroControls = ({ status, play, pause, stop, skip }) => (
-  <Container>
-    <PomodoroControl
-      icon='stop'
-      size='small'
-      design='tertiary'
-      onClick={stop}
-      disabled={Boolean(status === 'IDLE')}
-    />
-
-    {status === 'RUNNING' ? (
-      <PomodoroControl
-        icon='pause'
-        onClick={pause}
-        width='28'
-        height='28'
-      />
-    ) : (
-      <PomodoroControl
-        icon='play'
-        onClick={play}
-        width='38'
-        height='38'
-      />
-    )}
-
-    <PomodoroControl
-      icon='skip'
-      size='small'
-      design='tertiary'
-      onClick={skip}
-    />
-  </Container>
-)
+class PomodoroControls extends React.PureComponent {
+
+  constructor (props) {
+    super(props)
+    this.handleKeyboard = this.handleKeyboard.bind(this)
+  }
+
+  componentDidMount() {
+    document.addEventListener("keydown", this.handleKeyboard)
+  }
+
+  handleKeyboard = event => {
+    const { status, play, pause, stop, skip } = this.props
+    switch(event.keyCode){
+      case 13:
```

Comment:
would be nice to do something like putting the key codes in a hash or something like that
eg
```
const KeyboradKeys ={
 up: 12,
 space: 99
...
}
```
(I don't know the real number of the keys so probably 99 is not space 😂

Reply:
I can also use key.code in the switch case